### PR TITLE
RD-4376 Update blueprint tile

### PR DESCRIPTION
**Pull request by leon-good-life**
Jira ticket: https://cloudifysource.atlassian.net/browse/RD-4376

PR in Cloudify-Stage repo: https://github.com/cloudify-cosmo/cloudify-stage/pull/1863

### CODE REVIEW

File: content/working_with/console/widgets/blueprints.md [hunk 1]
```diff
@@ -129,6 +129,7 @@ See Settings section for details on how to turn on/off this feature.
 ## Settings
 
 * `Refresh time interval` - The time interval in which the widget’s data will be refreshed, in seconds. Default: 10 seconds
+* `List of fields to show` - The fileds that presented in the table and the catalog.
```
Comment (qooban) [inline content/working_with/console/widgets/blueprints.md:132]:
```suggestion
* `List of fields to show` - The fields that are presented in the blueprints list. Default: Created, Updated, Creator, State, Deployments
```

**Review by qooban — COMMENTED** [review]
1. We missed one more file - `content/working_with/console/widgets/blueprintCatalog.md` should be updated as well as the name of widget configuration field has changed.
2. When adding screenshots consider using https://compresspng.com/ - I checked that we can save 69% if we compress `static/images/ui/pages/local-blueprints-page.png`.

File: content/working_with/console/widgets/blueprints.md (hunk 1 above)
Comment (leon-good-life) [reply content/working_with/console/widgets/blueprints.md:132]: I think that the default are: Creator, State, Deployments.
When I click on "Reset Templates" those 3 fields are shown. And the task itself is to limit to those three.

Comment (leon-good-life) [reply content/working_with/console/widgets/blueprints.md]: I changed to:
```
* `List of fields to show` - The fields that are presented in the blueprints list. Default: Creator, State, Deployments.
```
[30a079d](https://github.com/cloudify-cosmo/docs.getcloudify.org/pull/2137/commits/30a079d7a8a627f00e03aeabbd4bb65a66e1079d)

Is it okay?

Comment (qooban) [reply content/working_with/console/widgets/blueprints.md]: No, you are talking about default for Blueprints page. And here we are talking about default for Blueprints widget.

**Comment by leon-good-life** [comment]
I updated BlueprintsCatalog.md: [ae25174](https://github.com/cloudify-cosmo/docs.getcloudify.org/pull/2137/commits/ae25174071bdc0bf65554f6adb9c68eb5a16109a)

File: content/working_with/console/widgets/blueprintCatalog.md [hunk 2]
```diff
@@ -25,6 +25,7 @@ These parameters are a must if you want to configure the widget to access privat
 ## Settings
 
 * `Blueprint Examples URL` - Specifies the JSON file from which the blueprints are being read
+* `List of fields to show` - The fields that are presented in the blueprints list. Default: Creator, State, Deployments.
```
Comment (qooban) [inline content/working_with/console/widgets/blueprintCatalog.md:28]: Have you checked the code?

It seems to me that the default is: Name, Description, Created, Updated.

File: content/working_with/console/widgets/blueprints.md (hunk 1 above)
Comment (leon-good-life) [reply content/working_with/console/widgets/blueprints.md]: I updated it: [cc9bc7b](https://github.com/cloudify-cosmo/docs.getcloudify.org/pull/2137/commits/cc9bc7b449fea83173573a2639d78e27decfdc59)

File: content/working_with/console/widgets/blueprintCatalog.md (hunk 2 above)
Comment (leon-good-life) [reply content/working_with/console/widgets/blueprintCatalog.md:28]: I checked right now, you are right:
Screen Shot 2022-03-23 at 10 09 03

Comment (leon-good-life) [reply content/working_with/console/widgets/blueprintCatalog.md]: I updated it: [27d7c93](https://github.com/cloudify-cosmo/docs.getcloudify.org/pull/2137/commits/27d7c93de07b9e2983ede97fff74af61423ecc42)

**Review by qooban — APPROVED** [review]

**Pull request merged**